sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -67,4 +67,15 @@ def compressed_no_header():
 @streams_api.route('/compressed', methods=['GET'])
 def compressed():
     return Response(stream_compressed_header_error(), status=300, headers={"Content-Encoding": "gzip"})
+
+def stream_decompress_header():
```

Comment:
Nitpick; I'd consider renaming this because what it really does/represents is a gzip content-encoded stream (iterable) of bytes.